src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -9,12 +9,18 @@ import scala.concurrent.Await
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration.Duration
 import scala.util.{Failure, Success, Try}
+import scala.annotation.tailrec
 
 /**
   * Entry point for synchronizing data between the Tezos blockchain and the Conseil database.
   */
 object Lorre extends App with LazyLogging {
 
+  private final val DefaultNetwork = "zeronet"
```

Comment:
It's dangerous to have a fallback as there might be an existing process handling `zeronet`. It's best to fail here if a valid network is not provided.

Reply:
fine for me

for the record: I just did refactor the code so that it wouldn't introduce any regression.